tests/sundials/kinsol_05.cc

```diff
@@ -49,6 +49,26 @@
 // function, which has the current iterate. But it converges slowly
 // because it doesn't update the Jacobian very often. This test
 // finally updates it in every iteration.
+//
+// In addition, this test checks that an optional custom_setup callback is
+// called correctly.
```

Comment:
Would you be willing to put this into a new test, rather than adding it to an existing one? We like to have tests where each test checks a separate component of deal.II, and this counts as a separate component. It's ok to just clone the existing one -- in other words, if you copy what you have into a new test, and leave the old one unchanged.